pom.xml

```diff
@@ -233,7 +233,11 @@
             <plugin>
                 <groupId>com.cosium.code</groupId>
                 <artifactId>git-code-format-maven-plugin</artifactId>
-                <version>2.5</version>
+                <version>2.6</version>
```

Comment:
this change isn't related to what's in the commit message

Reply:
Yes, as soon GitHub comes with the tooling to split commits, I am on it doing this little things... (or you pay me by hour)  ;-)